packages/0xcert-ethereum-utils-contracts/src/contracts/permission/abilitable.sol

```diff
@@ -102,35 +96,32 @@ contract Abilitable
     uint256 _abilities
   )
     external
-    hasAbilities(ABILITY_TO_MANAGE_ABILITIES)
```

Comment:
It's possible that in the future we'll have multiple super abilities! You should rename this const.

Reply:
If we do that, we can change it then.. Currently there is no projected change for multiple super abilities.